halo/app/app_config.go

```diff
@@ -132,14 +132,27 @@ var (
 		minttypes.ModuleName,
 	}
 
-	moduleAccPerms = []*authmodulev1.ModuleAccountPermission{
-		{Account: authtypes.FeeCollectorName},
-		{Account: distrtypes.ModuleName, Permissions: []string{authtypes.Burner}},
```

Comment:
previously distribution didn't have burner permission, this was added earlier today. Revert that and put it behind the feature flag.